sunpy/util/decorators.py

```diff
@@ -182,3 +183,28 @@ def deprecate(obj, message=message, name=name, alternative=alternative):
         return deprecate(message)
 
     return deprecate
+
+
+class add_common_docstring(object):
+    """
+    A function decorator that will append and/or prepend an addendum
+    to the docstring of the target function.
+
+    If a dictionary(d) is provided as parameter then format by .format().
+    """
+    def __init__(self, append='', prepend='', d={}):
```

Comment:
Would it be better if instead of `d={}` we changed this to `**kwargs` so all extra kwargs are passed to format?